app/urls.py

```diff
@@ -1,14 +1,12 @@
-
-# from os import stat
 from django.contrib import admin
 from django.urls import path, include
-
 from django.conf.urls.static import static
 from django.conf import settings
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('jobseeker.urls')),
+    path('profile/', include('personal_profile.urls')),
```

Comment:
You shouldn't edit this file, we have set all urls through the jobseeker app url file, so you only need to update the personal_profile app urls file as you did

Reply:
done